apps/web/app/api/health/route.ts

```diff
@@ -0,0 +1,9 @@
+import { NextRequest, NextResponse } from "next/server";
+
+export const GET = async (_req: NextRequest) => {
+  const webAppStatus = { status: "ok", message: "Web app is working" };
+
+  return NextResponse.json({
+    webApp: webAppStatus,
```

Comment:
Do you mind just inlining the status in the response? Something like:
```suggestion
    status: "ok",
    message: "All good"
```